Makefile

```diff
@@ -308,7 +310,9 @@ $(JSON_CRT_TICKET): clang.db
 
 $(JSON_TICKET): clang.db
 	mkdir -p obj/src/thread
-	repo-create-ticket --output=$@ --repo=$< 61823da085f534c947264e1497f73741
+	repo-create-ticket --output=obj/src/thread/__set_thread_area.t --repo=$< 61823da085f534c947264e1497f73741
+	repo-create-ticket --output=obj/src/setjmp/x86_64/longjmp.t --repo=$< b4969a1aad5e095bfdb567c8929359a2
```

Comment:
This should really be one rule per ticket file.

Reply:
Thanks, fixed in the commit https://github.com/SNSystems/musl-prepo/pull/9/commits/a434172d9f773d451c638755703f6814bc86e275.